groups/sig-k8s-infra/groups.yaml

```diff
@@ -203,6 +203,7 @@ groups:
       - cblecker@gmail.com
       - davanum@gmail.com
       - nikhitaraghunath@gmail.com
+      - wojtekt@google.com
```

Comment:
@wojtek-t why this is needed ? How the redirect is related to integration job failures ?

Reply:
I know that I'm able to see the logs of the failures but don't know which entry is the one that allows me to do that, so I suggested to add himself to the same groups I belong XD

Reply:
![image](https://user-images.githubusercontent.com/6450081/161218599-2c2f9119-ca62-481f-9cda-0b88fa623edf.png)

Reply:
Only membership of `k8s-infra-prow-viewers@kubernetes.io` is required. And @wojtek-t is supposed to be able to see metrics of this project. 
- https://github.com/kubernetes/k8s.io/blob/main/groups/sig-scalability/groups.yaml#L53-L72
- https://github.com/kubernetes/k8s.io/blob/b76a417a89b46ecc760c7a73472c2d46b39aa2f2/groups/sig-testing/groups.yaml#L144
https://console.cloud.google.com/monitoring/dashboards?project=k8s-infra-prow-build&pageState=(%22dashboards%22:(%22t%22:%22All%22))

Reply:
This is super strange...

I've seen that, but I was trying to access that project an hour ago and didn't have access.
But I tried now and I have access - I have no idea what happened.

Anyway - closing this PR for now.